### PR TITLE
content pages using rST (e.g. release announcements)

### PR DESCRIPTION
**Pull request by enkore**

### CODE REVIEW

File: releases/index.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Borg 1.0
+--------
+
+The current **stable** series is Borg 1.0, with the initial 1.0.0 release on 5th March 2016.
+The latest release in this series is Borg 1.0.10, released on 13th February 2017.
+This is the series shipped by most Linux distributions.
+
+- `Documentation <https://borgbackup.readthedocs.io/en/stable/>`__
+- `Installation <https://borgbackup.readthedocs.io/en/stable/installation.html>`__
+- `Downloads <https://github.com/borgbackup/borg/releases/latest>`__
```
Comment (ThomasWaldmann) [inline releases/index.rst]: that will only work until we do a 1.1 release.

**Comment by ThomasWaldmann** [comment]
Nothing against having some generic / long term content like you did (0.x, 1.0, 1.1), but I think we should avoid putting individual release notes there - it would be just a duplication of changelog / of what's visible on gh release page.

File: releases/index.rst (hunk 1 above)
Comment (enkore) [inline releases/index.rst]: Yes, these will need to be shuffled around when the stable series changes. 1.1 latest->stable, 1.0 stable->1.0-stable (? 1.0-maint is IIRC the unreleased branch).

**Comment by enkore** [comment]
I haven't made up my mind on that yet, whether RAs (including summarized change logs) should go only into the release docs, only on a separate page (seems like the most common thing with change-intensive projects, a big changelog with everything and a separate location with RAs that summarize changes and mention contributors), or both (duplication in docs is often good, but 1:1 copies seem not very useful).

File: releases/index.rst (hunk 1 above)
Comment (enkore) [inline releases/index.rst]: Hmm, no, that doesn't really work.

**Pull request merged**